Snowdog_Components/components/Organisms/mega-menu/mega-menu.hbs

```diff
@@ -1,5 +1,5 @@
-<div class="mega-menu">
-    <ul class="mega-menu__list container">
+<div class="mega-menu {{ class }}" {{{ attributes }}}>
+    <ul class="mega-menu__list container {{ listClass }}">
```

Comment:
should be `classList`, like you've declared in config

Reply:
thanks. I fixed name in config to follow convention (like in breadcrumbs)